package.json

```diff
@@ -87,15 +87,7 @@
     "lint:fix": "npm run lint -- --fix",
     "lint": "eslint . --cache",
     "lint-staged": "lint-staged",
-    "release:major": "env SEMANTIC=major npm run release",
-    "release:minor": "env SEMANTIC=minor npm run release",
-    "release:patch": "env SEMANTIC=patch npm run release",
-    "replace:versions": "replace-x -r 'uppy/v\\d+\\.\\d+\\.\\d+/dist' \"uppy/v$npm_package_version/dist\" ./examples/ README.md bin/upload-to-cdn.sh website/src/examples/ website/src/docs/ website/themes/uppy/layout/ --exclude=node_modules",
-    "replace:versions:commit": "git commit -m \"Change Uppy version references to v$npm_package_version\" ./examples/ README.md bin/upload-to-cdn.sh website/src/examples/ website/src/docs/ website/themes/uppy/layout/",
-    "release:tag": "npm version ${SEMANTIC:-patch} -m \"Release %s\"",
-    "release:build": "npm-run-all clean build release:copy-uppy-readme",
-    "release:copy-uppy-readme": "cp README.md packages/uppy/README.md",
-    "release": "npm-run-all release:tag release:build replace:versions replace:versions:commit && git push && git push --tags && npm publish",
+    "release": "./bin/release",
     "start:server": "node bin/start-server",
```

Comment:
Soooo, how do we release a patch or minor now? :)

Reply:
lerna will ask what to do for each package